### DISCUSSION TITLE
Google\\ApiCore\\ApiException(code: 3)

### DISCUSSION THREAD

**Discussion by angryalf** [stackkit/laravel-google-cloud-tasks-queue#156] [General]
I used Shopify app, inside webhook controller I got error with dispatching job.. any ideas?

{
    "message": "HttpRequest.url must start with 'https:\/\/' for request with HttpRequest.authorization_header",
    "code": 3,
    "status": "INVALID_ARGUMENT",
    "details": []
} {"exception":"[object] (Google\\ApiCore\\ApiException(code: 3): {
    \"message\": \"HttpRequest.url must start with 'https:\\/\\/' for request with HttpRequest.authorization_header\",
    \"code\": 3,
    \"status\": \"INVALID_ARGUMENT\",
    \"details\": []
}

**Comment by marickvantuil** [comment]
Apologies I assume you have already solved the issue, but to anyone, this happens likely because the application runs behind an untrusted proxy. Can be fixed by setting the trusted proxy: https://laravel.com/docs/12.x/requests#trusting-all-proxies
